### ISSUE TITLE
Definition of gist:Building is too broad

### ISSUE THREAD

**Issue by rjyounes** [semanticarts/gist#482]
Definition is "A man-made structure for dwelling or working."

According to this, construction scaffolds, parking lots (where a parking lot attendant works),  elevators, and food trucks, among many other things, are all buildings.

**Comment by uscholdm** [comment]
Good point. Maybe try a dictionary or Wikipedia for a better definition? Or a school of architecture?

**Comment by rjyounes** [comment]
Look at dictionary definitions.

**Comment by rjyounes** [comment]
But see #1205.

**Comment by stevenchalem** [comment]
Closing as completed; the definition was re-written since this issue was opened.
